src/index.html

```diff
@@ -7,6 +7,100 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar__month-length--29 calendar__start-day--mon">
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="day"></div>
+      </div>
```

Comment:
Don't create redundant divs, use only one wrapper for each day
```suggestion
      <div class="calendar__day"></div>
```